templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -160,12 +192,28 @@ Resources:
   VPCStack:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template
-        - QSS3Region: !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+      TemplateURL: !If
+        # Historically this template has never been supplied with the parameters; QSS3BucketName or QSS3KeyPrefix therefore it
+        # has ALWAYS been interpolated (regardless of product stack) to its default value which in this case is:
+        # http://aws-quickstart.s3.amazonaws.com/quickstart-atlassian-services/submodules/quickstart-aws-vpc/templates/aws-vpc.template
+        #
+        # This is also the case for templates sourced from custom S3 buckets! 
+        # 
+        # As a result if we change this template we need to make sure it will accommodate those product templates which are being called 
+        # from custom S3 buckets and do not pass the parameters; QSS3BucketName, QSS3KeyPrefix. If we dont do this stack deployments will 
+        # fail. As a result we keep the original yaml below to fall back to in the event that this template is called without the parameters 
+        # mentioned
```

Comment:
Consider in code comment, a PR comment. It will be removed once we get agreement on why this logic is here. Nasty stuff, although it should make us backward compatible and prevent existing customers using their their own buckets running into issues